package/src/main.scss

```diff
@@ -123,4 +123,64 @@ $control-color-stop: #f55;
       box-shadow: 0 1em 0 0;
     }
   }
+
+  // Plyr overrides
+  .plyr--audio {
+    width: 100%;
+
+    .plyr__controls {
+      padding: 0;
+      background: transparent;
+
+      .plyr__controls__item {
+        margin-left: 0.5rem;
+      }
+    }
+    .plyr__control {
+      &:hover,
+      &.plyr__tab-focus,
+      &[aria-expanded="true"] {
+        border-color: $control-color-default;
+        background-color: $control-background-color;
+        color: $control-color-default;
+      }
+    }
+    // Overlaid "play" button used for videos
+    .plyr__control--overlaid {
```

Comment:
Strictly this only make sense in a theme file which overrides the the standard behavior of plyr.
Difficult to know if this selector will bind strongly enough for the intention of this block...